fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dto/StringParmVal.java

```diff
@@ -11,27 +11,21 @@
 /**
  * This class defines the Data Transfer Object representing a row in the X_STR_VALUES tables.
  */
-public class StringParmVal implements ExtractedParameterValue {
-    
-    private String resourceType;
-    private String name;
+public class StringParmVal extends ExtractedParameterValue {
+
+    // The string value of this extracted parameter
     private String valueString;
-    
+
     // The SearchParameter base type. If "Resource", then this is a Resource-level attribute
```

Comment:
Use base already defined in ExtractedParameterValue?

Reply:
removed